tests/main/i18n/task.yaml

```diff
@@ -13,4 +13,11 @@ execute: |
     fi
 
     echo "Ensure that i18n works"
-    LANG=de_DE.UTF-8 snap changes everything | MATCH "Ja, ja, allerdings."
+    LANG=de snap changes everything | MATCH "Ja, ja, allerdings."
+
+    echo "Basic smoke test to ensure no locale causes crashes"
+    for p in /usr/share/locale/*; do
+        export LANG=$(basename $p)
```

Comment:
I think you may need something like `language-support-XX` for this to work

Reply:
While true in the general case, we use our own i18n native go code which bypasses the need for language-support-XX .